packages/cmf/README.md

```diff
@@ -36,29 +36,34 @@ It fits with our goal, this is why this add-on has been named that way.
 
 ## Paradigm
 
-A _user_ interacts with a _view_ using mouse and/or keyboard which sends _events_ from a _content_ and that interaction _dispatches_ an _action_.
-That action may change the current view or the content displayed.
+A _user_ interacts with a _component_ using mouse and/or keyboard which sends _events_ from a _content_ and that interaction _dispatches_ an _action_.
+That action may change the current component or the content displayed.
```

Comment:
not 100% true
in fact that action may change the application state, wish in turn may or may not impact said component